### DISCUSSION TITLE
I want to start a single postgrest server where i can query with multiple DB.

### DISCUSSION THREAD

**Discussion by vaibhavshah08** [PostgREST/postgrest#3047] [Q&A] [answered]
### Description of issue
Basically I need to add the multiple Schemas in the single postgrest and i need to query with in a single schema. 
Basically I want to achieve this 
Lets talk about tenant based queries 
lets take and example like in my database their are 5 tenants (nothing but schema) and now i want that tenant 1 should not have access to other tenants and same for every tenant so to achieve this case how to use postrest. I need authentication and authorization also for every tenant

**Comment by wolfgangwalther** [comment, marked as answer]
For a tenant based setup, you should look into [Row Level Security](https://www.postgresql.org/docs/current/ddl-rowsecurity.html). This can be combined with [SQL User Management](https://postgrest.org/en/stable/how-tos/sql-user-management.html).
